packages/atomic/src/pages/index.html

```diff
@@ -162,6 +162,7 @@
         <atomic-date-facet field="date" label="Date"></atomic-date-facet>
         <atomic-category-facet field="geographicalhierarchy" label="World Atlas" enable-facet-search>
         </atomic-category-facet>
+        <atomic-rating-facet field="size" label="Rating"> </atomic-rating-facet>
```

Comment:
When testing with `npm run dev`, the atomic rating facet appears but has no facet-values. I tried with both fields `size` and `rating`.